app/retail/templates/emails/grants/grant_cancellation.html

```diff
@@ -1,46 +1,44 @@
-{% extends 'emails/template.html' %}
```

Comment:
No reason for the trailing whitespace additions here.
All vars need updated to use the input grant object, none of these variables will exist.
Follow django proper whitespace for template variables and functions.
URLs need updated to use the `url` reverse.
We need all of these templates hooked up to the respective email func and passing the grant and/or subscription objects into the email methods on event.